router/pkg/config/config.schema.json

```diff
@@ -519,6 +519,11 @@
           "default": true,
           "description": "The allowed credentials. The default value is to allow credentials. This allows the browser to send cookies and authentication headers."
         },
+        "allow_wildcard": {
```

Comment:
@YassineElbouchaibi You also need to pass it to `core.WithTLSConfig` in `instance.go` otherwise the setting is not applied. How did you test it?

Reply:
Ah yes sorry missed that... I have added it to `core.WithCors` now. I did not test unfortunately as I have made these changes directly in the web and I don't have a working local setup. I was hoping the pipeline tests would pickup anything wrong. Let me know if you think something else is missing.

Reply:
I have approved the run. After the build, you can test the image with

`docker pull ghcr.io/wundergraph/cosmo/router:pr-880`

Would that work for you?

Reply:
Yes I definitely can test that, thanks!

Reply:
Ideally, you can add an integration test to https://github.com/wundergraph/cosmo/tree/main/router-tests but since we did not have any tests for CORS, I can help you with that (if needed). Let me know what you think.

Reply:
My bad. Images from contributors aren't pushed due to insufficient permissions. You can build the image locally with `cd router && docker build -t router:latest .`

Reply:
Sorry for the delay, my local setup was not able to build the image locally because of the following:

<img width="960" alt="image" src="https://github.com/wundergraph/cosmo/assets/46449697/1ff4ae52-2bcb-4b4a-a094-5bfa437537bf">

so I had to go through various hoops to get it working..

But I can now confirm wild card origins work and stop working when the setting is changed to `false`.

Regarding integration tests, It will be hard for me to find time for it.. I'm not very familiar with golang let alone testing in golang.. but if you give me enough time I can look into it.